geomesa-tools/bin/geomesa-kafka

```diff
@@ -247,5 +247,5 @@ else
         shift 1
     fi
     CLASSPATH="$GEOMESA_CP:$ACCUMULO_CP:$HADOOP_CP"
-    java ${GEOMESA_OPTS} -cp ${CLASSPATH} org.locationtech.geomesa.tools.Runner "$@"
+    java ${GEOMESA_OPTS} -cp ${CLASSPATH} org.locationtech.geomesa.tools.kafka.KafkaRunner "$@"
```

Comment:
I'd suggest keeping 'geomesa' as an Accumulo-based command for the duration of the GeoMesa 1.2 series.  

As such, copying bin/geomesa to geomesa-kafka makes sense, but a copy of bin/geomesa would remain.

@jahhulbert-ccri  @elahrvivaz @cne1x thoughts?